README.md

```diff
@@ -14,6 +14,8 @@
 ## What is SPV-Wallet?
 [Read more about SPV-Wallet](https://replace-after-moving-spv-wallet)
 
+[SPV Wallet GitBook](https://bsvblockchain.gitbook.io/docs)
```

Comment:
IMHO this link should be introduced with some text, e.g. what it is and what an user can find there.